packages/composite-checkout/src/components/checkout-payment-methods.js

```diff
@@ -30,17 +30,9 @@ export default function CheckoutPaymentMethods( {
 		: paymentMethods;
 
 	if ( summary && isComplete && paymentMethod ) {
-		return (
-			<div className={ joinClasses( [ className, 'checkout-payment-methods' ] ) }>
-				<PaymentMethod
-					{ ...paymentMethod }
-					checked={ true }
-					summary
-					ariaLabel={ getAriaLabelForPaymentMethodSelector( paymentMethod.id, localize ) }
-				/>
-			</div>
-		);
+		return localize( paymentMethod.summaryText );
```

Comment:
@sirbrillig any ideas about how we might store and capture the data from our payment step so we can show it here?

Reply:
@sirbrillig I gave this another shot using the data object. It currently handles the regular credit card. We will have to deal with stripe separately.